pkg/assets/bindata.go

```diff
@@ -231,37 +231,6 @@ func config_manager_metrics_service_yaml() ([]byte, error) {
 	return _config_manager_metrics_service_yaml, nil
 }
 
-var _config_manager_namespace_yaml = []byte(`apiVersion: v1
-kind: Namespace
-metadata:
-  annotations:
-    openshift.io/node-selector: ""
-  labels:
-    controller-tools.k8s.io: "1.0"
-    openshift.io/run-level: "1"
-    openshift.io/cluster-monitoring: "true"
-  name: openshift-cloud-credential-operator
-`)
-
-func config_manager_namespace_yaml() ([]byte, error) {
-	return _config_manager_namespace_yaml, nil
-}
-
-var _config_manager_operator_configmap_yaml = []byte(`apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cloud-credential-operator-config
-  namespace: openshift-cloud-credential-operator
-  annotations:
-    release.openshift.io/create-only: "true"
-data:
-  disabled: "true"
-`)
-
-func config_manager_operator_configmap_yaml() ([]byte, error) {
-	return _config_manager_operator_configmap_yaml, nil
-}
-
```

Comment:
This looks suspicious... this is fairly new stuff being removed, wondering if there was a problem during devel on an older branch that leaked into your current PR?

Reply:
I cloned it today and I'm rebased on top of your merged commit 9b71397a.
What version of mockgen should I use?

Reply:
it looks like on my machine i have golang-github-mock-1.2.0-3.fc31.x86_64 installed

Reply:
Thanks, that probably work, the PR is updated.

Do you want to document the mockgen version in the README?

Reply:
i think this might be happening for a similar reason to why i had to make the second commit in PR #132 . once 132 merges, this PR should just need a simple rebase